bigtable/client/idempotent_mutation_policy.cc

```diff
@@ -14,6 +14,8 @@
 
 #include "bigtable/client/idempotent_mutation_policy.h"
 
+#include <ciso646>
```

Comment:
It might be easier to move this `#include` to `bigtable/client/version.h` and then everybody gets it by default.